wpilibj/src/athena/java/edu/wpi/first/wpilibj/PWM.java

```diff
@@ -124,7 +90,7 @@ public void free() {
     if (m_handle == 0) {
       return;
     }
-    PWMJNI.setPWM(m_handle, (short) 0);
+    setDisabled();
```

Comment:
Its the same behavior. The old setPWM set the raw value, and 0 was kPwmDisabled. For some reason it just didn't use the constant.